includes/testimonial-list.html

```diff
@@ -0,0 +1,71 @@
+{%- if include.json -%}
+  {%- assign testimonialListHeadline = include.json.headline -%}
+  {%- assign testimonialListHeadlineLevel = include.json.headlineLevel -%}
+  {%- assign testimonialListSubline = include.json.subline -%}
+  {%- assign testimonialListContents = include.contents | include.json.contents -%}
+  {%- assign testimonialBgColor = include.json.bgColor -%}
+  {%- assign testimonialBgColorHover = include.json.bgColorHover -%}
+{%- else -%}
+  {%- assign testimonialListHeadline = include.headline -%}
+  {%- assign testimonialListHeadlineLevel = include.headlineLevel -%}
+  {%- assign testimonialListSubline = include.subline -%}
+  {%- assign testimonialListContents = include.contents -%}
+  {%- assign testimonialBgColor = include.bgColor -%}
+  {%- assign testimonialBgColorHover = include.bgColorHover -%}
+{%- endif -%}
+{% assign testimonialListEven = false %}
+
+<!-- vue commponent -->
+<testimonial-list
+  testimonials="{{ testimonialListContents | jsonify | escape }}"
+>
+  {% assign testimonialListSize = testimonialListContents | size %}
+  {% assign testimonialListEven = testimonialListSize | modulo: 2 %}
+  {% for testimonial in  testimonialListContents %}
```

Comment:
Hier würde ich z.B. eine Leerzeile vor der for Schleife machen. In Zeile 39 hast du dann eine Leerzeile, die meiner Meinung dort nix bringt und 46-47 dann 2 :D sieht man keinen roten Faden.